pages/static-security-analysis.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Static Security Analysis
+permalink: /static-security-analysis/
+---
+
+Static analysis is an important part of the development process, and is required for ATO. There are two main types of static security testing that needs to be done: dependency analysis, where the ruby gems, python modules, and js packages your app uses are checked against a list of known vulnerabilities; and code analysis in which your code is checked against a list of antipatterns. There are tools for JS, Ruby, and Python, and you are encouraged to set up this scanning early on in the development cycle to prevent unexpected delays when it's time to get your ATO.
+
+## Javascript
+
+### Dependency analysis
+
+We recommend using the Node Security Project's [NSP tool](https://nodesecurity.io/tools) to scan your package.json or npm-shrinkwrap.json file for known security issues.
+
+To install:
+
+    $ npm install nsp
```

Comment:
I buy that, but I think we will likely be going beyond the basics for some tools so for completeness' sake I was including all of them.